apps/admin/frontend/src/contexts/services_context.ts

```diff
@@ -1,25 +1,16 @@
 import { Logger, LogSource } from '@votingworks/logging';
-import { MemoryStorage, Storage } from '@votingworks/utils';
 import { createContext } from 'react';
-import {
-  ElectionManagerStoreBackend,
-  ElectionManagerStoreMemoryBackend,
-} from '../lib/backends';
 
 /**
  * Provides access to various services.
  */
 export interface ServicesContextInterface {
-  readonly backend: ElectionManagerStoreBackend;
   readonly logger: Logger;
-  readonly storage: Storage;
 }
```

Comment:
Now that this is just the logger I will, in a future commit, remove this additional context and place the logger in the AppContext. But wanted to keep any avoidable churn outside this PR.